.github/workflows/build.yaml

```diff
@@ -4,8 +4,8 @@ on:
   workflow_dispatch:
   pull_request:
   push:
-    branches: ["*"]
-    tags: ["*"]
+    branches: ["main", "release/*", "project/*"]
+    tags: ["v*"]
```

Comment:
I'm not so sure about this tag filter. Our `release` job runs [when the tag starts with `"Second_Life_"`](https://github.com/secondlife/viewer/blob/main/.github/workflows/build.yaml#L329). I think this change would keep such tag builds from running at all, preventing us from getting viewer release builds.

I don't mind the `branches` restriction.

Reply:
Alright, I've opened the tag trigger back up to `*`.

Reply:
Thanks. I don't think we'll get a lot of spurious tag builds anyway. I suppose we could say

    tags: ["Second_Life_*"]

if we want to suppress tag builds for the (e.g.) "7.1.2-release" tags.

Reply:
Done